charts/longhorn/100.0.0+up1.1.2/Chart.yaml

```diff
@@ -2,10 +2,11 @@ annotations:
   catalog.cattle.io/auto-install: longhorn-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Longhorn
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.22.0-0'
   catalog.cattle.io/namespace: longhorn-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
-  catalog.cattle.io/rancher-version: '>= 2.6.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.6.0-0 < 2.7.0-0'
```

Comment:
Does it mean that we will update this field again when rancher release v2.7.0? Because this chart will still be compatible with the new rancher release

Reply:
Yes, because in 2.7 we will most likely cut a new release-v2.7 branch for all charts, bump the versions to 200.x.x where it applies, and bump this to `'>= 2.7.0-0 < 2.8.0-0'`.